AdyenCard/Components/Card/CardViewController.swift

```diff
@@ -198,11 +198,26 @@ internal class CardViewController: FormViewController {
 
     internal func update(binInfo: BinLookupResponse) {
         var brands: [CardBrand] = []
+
         // no dual branding if response is from regex (fallback)
         if binInfo.isCreatedLocally, let firstBrand = binInfo.brands?.first {
             brands = [firstBrand]
+            items.coBadgedCardItem.isHidden.wrappedValue = true
+            items.coBadgedCardItem.selectableFormItems = []
         } else {
             brands = binInfo.brands ?? []
+            if brands.count == 2, brands.allSatisfy(\.isSupported) {
```

Comment:
This if else block logic seems like it can be moved into the `coBadgedCardItem` itself. Similar to cardNumberItem handling it dual brand logic instead of here.

Reply:
Agree with Eren here